lib/puppetserver/ca/config/puppet.rb

```diff
@@ -126,9 +126,8 @@ def resolve_settings(overrides = {})
             :ca_server => '$server',
             :ca_port => '$masterport',
             :localcacert => '$certdir/ca.pem',
-            :localcacrl => '$ssldir/crl.pem',
```

Comment:
removing localcacrl as this is the same as hostcrl.

Reply:
good catch, it looks like it's not even a setting https://puppet.com/docs/puppet/5.5/configuration.html

Reply:
My b. Shoulda known better than to assume a naming pattern in Puppet settings :P